crates/subspace-farmer/README.md

```diff
@@ -17,21 +17,21 @@ enabled, it is by default), also LLVM/Clang is necessary.
 ### Ubuntu
 
 ```bash
-sudo apt-get install automake libtool pkg-config llvm clang protobuf-compiler
+sudo apt-get install automake libtool pkg-config llvm protobuf-compiler
 ```
 
 ### macOS
 
 1. Install via Homebrew:
 
 ```bash
-brew install automake libtool llvm clang protobuf
+brew install automake libtool llvm protobuf
 ```
 
 2. Add `llvm` to your `~/.zshrc` or `~/.bashrc`:
 
 ```bash
-export PATH="/opt/homebrew/opt/llvm@15/bin:$PATH"
+export PATH="/opt/homebrew/opt/llvm@17/bin:$PATH"
```

Comment:
Are you sure `@17` is even necessary? IIRC we needed `@15` when newer version of LLVM wasn't supported, but now that it does, I'm wondering if there is just `llvm` directory or symlink there.

Reply:
That is exactly my point. Today it is LLVM 17, tomorrow LLVM 18, so why would we hardcode version in there.